shared/my-sites/themes/main.jsx

```diff
@@ -67,7 +71,7 @@ var Themes = React.createClass( {
 	togglePreview: function( theme ) {
 		const site = this.props.sites.getSelectedSite();
 		if ( site.jetpack ) {
-			Action.customize( theme, site );
+			this.props.dispatch( Action.customize( theme, site ) );
```

Comment:
> OTOH, adding reduxStore to window isn't going to play nice with SSR :-/

That was my first reaction. Passing `dispatch` here seems good enough™ to me, for a first iteration at least.